docs/UIDS-GIDS.md

```diff
@@ -263,7 +278,8 @@ i.e. somewhere below `/var/` or similar.
 |            1000…60000 | 0x000003E8…0x00001770 |      59000 | Regular users                     | Distributions | `/etc/passwd` + LDAP/NIS/…    |
 |           60001…60513 | 0x0000EA61…0x0000EC61 |        513 | Human users (homed)               | `systemd`     | `nss-systemd`                 |
 |           60514…60577 | 0x0000EC62…0x0000ECA1 |         64 | Host users mapped into containers | `systemd`     | `systemd-nspawn`              |
-|           60578…61183 | 0x0000ECA2…0x0000EEFF |        606 | *unused*                          |               |                               |
+|           60578…61089 | 0x0000ECA2…0x0000EEA1 |        512 | Dynamic greeter users             | `systemd`     | `nss-systemd`                 |
```

Comment:
That's a lot of the free ones, does this feature really need 512 greeters at the same time?

Reply:
Yeah it's some headroom but not unreasonable. A server serving remote desktop clients could have lots of greeters running

Happy to shrink the range though. 256 or even 128 seem reasonable to me too

Reply:
Maybe start with 128, and if needed it can be expanded later? Increasing the range can be done without breaking compat